data/filters/youtube-cleanup.yaml

```diff
@@ -8,6 +8,14 @@ params:
     description: Hide the text next to the dislike/share/download/report/save buttons below a video.
     type: checkbox
     default: false
+  - name: remove-comment-section
+    description: Hide the comment section below a video.
+    type: checkbox
+    default: false
+  - name: hashtags
```

Comment:
nitpick: let's keep it consistent with the naming of the options options, maybe `remove-video-hashtags`?

Reply:
See https://github.com/xvello/letsblockit/pull/153/commits/d55073975515b68f21a962c3c5c76810a5f0fa9e